.github/workflows/create_release.yml

```diff
@@ -15,7 +15,7 @@ jobs:
   build:
     strategy:
       matrix:
-        runs-on: [ubuntu-latest, macos-13, macos-13-xlarge]
+        runs-on: [ubuntu-latest]
```

Comment:
We weren't using the MacOS binaries as official build artifacts, anyway:

https://github.com/multiversx/mx-chain-go/releases